engine/gamesys/src/gamesys/resources/res_glyph_bank.cpp

```diff
@@ -33,13 +50,19 @@ namespace dmGameSystem
     {
         GlyphBankResource* resource   = new GlyphBankResource();
         resource->m_DDF               = (dmRenderDDF::GlyphBank*) params->m_PreloadData;
+        resource->m_Font              = CreateGlyphBankFont(params->m_Filename, resource->m_DDF);
```

Comment:
A good outcome, being able to wrap the old, fixed font format in  the same api as the TTF fonts, making the code much more stream lined.